vtexdocs/devportal
# chore: update rapidoc submodule

#### What is the purpose of this pull request?

To fix the parameters table in the response component in API Reference pages.

#### What problem is this solving?

Table content overflowing.

#### How should this be manually tested?

Open API Reference pages and check the parameters table in the response component.

#### Screenshots or example usage

| Before | After |
|---|---|
|![image](https://github.com/vtexdocs/devportal/assets/62757720/6637825e-7a9b-46d0-ba20-f5b221056045)| ![image](https://github.com/vtexdocs/devportal/assets/62757720/9f83a9ae-5f29-4b9b-8c12-be3cdf56ee80)|

#### Types of changes
